src/views/Bond/components/BondList.tsx

```diff
@@ -193,19 +192,13 @@ const BondTable: React.FC<{ isInverseBond: boolean }> = ({ children, isInverseBo
 );
 const quoteTokenCapacity = (bond: Bond, isInverseBond: boolean) => {
   const quoteTokenCapacity = `
-  ${(bond.maxPayout.inQuoteToken.lt(bond.capacity.inQuoteToken)
-    ? bond.maxPayout.inQuoteToken
-    : bond.capacity.inQuoteToken
-  ).toString({ decimals: 3, format: true })}${" "}
+  ${bond.capacity.inQuoteToken.toString({ decimals: 3, format: true })}${" "}
   ${bond.quoteToken.name}`;
   return quoteTokenCapacity;
 };
 const payoutTokenCapacity = (bond: Bond, isInverseBond: boolean) => {
   const payoutFormatter = Intl.NumberFormat("en", { notation: "compact" });
-  const payoutTokenCapacity = `${(bond.maxPayout.inBaseToken.lt(bond.capacity.inBaseToken)
-    ? bond.maxPayout.inBaseToken
-    : bond.capacity.inBaseToken
-  ).toString()}`;
+  const payoutTokenCapacity = `${bond.capacity.inBaseToken.toString()}`;
```

Comment:
isn't `maxPayout` the governing number? if `maxPayout` is less than `capacity` then I think only `maxPayout can get paid.

Reply:
> isn't `maxPayout` the governing number? if `maxPayout` is less than `capacity` then I think only `maxPayout can get paid.

It's the inverse. if capacity is less than maxPayout, then your maxPayout would be what the remaining capacity is.

Reply:
otherwise your max payout would be what the maxPayout value is. In this scenario 64k.